tools/mason/README.rst

```diff
@@ -20,25 +20,18 @@ In the mason directory found in ``$CHPL_HOME/tools/mason`` run the following:
    make
    make install
 
-This will create the hidden ``.mason/`` directory in ``MASON_HOME``, which
-defaults to your ``$HOME`` if unset. It builds the mason binary so that the
-command line interface can be used. The mason registry is also downloaded from
-github so that a user can start to rely on mason to specify project
-dependencies. This installs mason in the same place as the chapel compiler (``chpl``)
-so that mason can be used anywhere in the user's file system.
+It builds the mason binary so that the command line interface can be used. This
+installs mason in the same place as the chapel compiler (``chpl``) so that
+mason can be used anywhere in the user's file system.
 
 
+The ``mason update`` command will download the registry to ``$MASON_HOME/registry``
+if it there is not already a registry at that location.
+
```

Comment:
Would this be a good place to state the minimum `git` version supported? I guess determining that version is a task in itself..

Reply:
Probably, but I fixed the "git -C" issue so I think we can punt on it for now.